Modix.Services/BehaviourConfiguration/BehaviourConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Modix.Services.BehaviourConfiguration
+{
+    public interface IBehaviourConfiguration
+    {
+        InvitePurgeBehaviour InvitePurgeBehaviour { get; set; }
```

Comment:
I think it'd be better to just have a generic `GetBehaviourConfigurationsAsync(BehaviourCategory)` method, and let individual features handle the conversion to an object type, if desired. 

I think doing it this way invites more interface coupling than we really need. I mean, who actually would ever need this property, other than `InvitePurgeHandler` objects? I'd argue that if any other services, not related to invite purging decide they want to interact with this, they actually SHOULDN'T be able to, and should be forced to refactor.